### PR TITLE
Add Documentation

### PR DESCRIPTION
**Pull request by b10n1k**
Add main structure for hosting and create documentation

qemetrics_doc_index_screenshoot

qemetrics_doc_modules_screenshoot

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+project = 'QE Metrics'
+copyright = '2023, ilausuch'
```
Comment (ilausuch) [inline docs/conf.py:15]: Let's set a opensource copyright: Apache, Mit, not sure

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+project = 'QE Metrics'
```
Comment (ilausuch) [inline docs/conf.py:14]: Let's call it Project Management Statistics

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+copyright = '2023, ilausuch'
+author = 'ilausuch'
```
Comment (ilausuch) [inline docs/conf.py:16]: Could be more than one author? will be others contributors

File: docs/conf.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+release = '1.0.0'
```
Comment (ilausuch) [inline docs/conf.py:17]: Ok, we don't have a roadmap, let's say alpha1?

**Comment by ilausuch** [comment]
Good job! some small things

**Comment by ilausuch** [comment]
Are the __init__.py required? maybe we can add them to the .gitignore

File: docs/conf.py (hunk 1 above)
Comment (b10n1k) [reply docs/conf.py:15]: this is a different variable. i can added

File: docs/conf.py (hunk 2 above)
Comment (b10n1k) [reply docs/conf.py:14]: Sure

File: docs/conf.py (hunk 3 above)
Comment (b10n1k) [reply docs/conf.py:16]: yes

Comment (b10n1k) [reply docs/conf.py:16]: but i put just you as it is mainly your modules so far

File: docs/conf.py (hunk 4 above)
Comment (b10n1k) [reply docs/conf.py:17]: i am not sure about that. maybe we can discuss it with Jan in some point

**Comment by b10n1k** [comment]
> Are the **init**.py required? maybe we can add them to the .gitignore

As per our offline discussion. This is no sense. __init__.py cant be in .gitignore as python uses them internally

File: docs/conf.py (hunk 1 above)
Comment (b10n1k) [reply docs/conf.py]: Actually this is not taking the Licence. So i used `SUSE LLT` as we do for os-autoinst

File: docs/conf.py (hunk 2 above)
Comment (b10n1k) [reply docs/conf.py]: Done

File: docs/conf.py (hunk 4 above)
Comment (b10n1k) [reply docs/conf.py]: I add version as `0-alpha1` and adjust release to reflect the major version

**Review by ilausuch — APPROVED** [review]

**Comment by ilausuch** [comment]
I am thinking we also could add a test in the workflow to check the documentation is generated well. With this this PR will be complete

**Pull request merged**